README.md

```diff
@@ -144,11 +144,11 @@ Users who need advanced customization or features should see the [User Manual](.
 
 ## Reporting Issues
 
-If you've found a bug or need support, the best way to get support is to email [support@fossa.com](mailto:support@fossa.com).
+If you've found a bug or need support, the best way to get support is via [FOSSA support portal](https://support.fossa.com/).
```

Comment:
Please prepend all these with "the":
```suggestion
If you've found a bug or need support, the best way to get support is via the [FOSSA support portal](https://support.fossa.com/).
```

Reply:
resolved with: https://github.com/fossas/fossa-cli/pull/1060/commits/8c30d2f8b59545b472b859a0b1df3d7462b9974e, https://github.com/fossas/fossa-cli/pull/1060/commits/1f997a7f4f42bfd39fe764167c56529ec6037d2c, https://github.com/fossas/fossa-cli/pull/1060/commits/b17339298733502fd756f19460e19fcbe3079377